modules/library/referencing/src/main/java/org/geotools/referencing/operation/transform/ThinPlateSpline2D.java

```diff
@@ -0,0 +1,139 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2002-2025, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.referencing.operation.transform;
+
```

Comment:
Would be good to add some javdoc and a reference, something like below..

```suggestion
/**
 * The a {@linkplain MathTransform2D two-dimensional transform} thin plate spline transform. 
 * 
 * <p>
 * References:
 * <ul>
 * <li><a href="https://en.wikipedia.org/wiki/Thin_plate_spline">Thin plate spline (Wikipedia)</a>
 * </ul>
 */
```

possibly you have other references as well